src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/SourceGenerators/RazorSourceGenerator.RazorProviders.cs

```diff
@@ -44,6 +44,10 @@ public partial class RazorSourceGenerator
 
             var razorConfiguration = new RazorConfiguration(razorLanguageVersion, configurationName ?? "default", Extensions: [], UseConsolidatedMvcViews: true);
 
+            // We use the new tokenizer by default
+            var useRazorTokenizer = !parseOptions.Features.TryGetValue("use-razor-tokenizer", out var useRazorTokenizerValue)
```

Comment:
I'm not sure the best way to actually test this. We could wait for the new tokenizer and test a side effect, like strings being handled better, but that doesn't seem like a great approach. I don't have any better ideas though.